test/schemas/v0/tests.schema.json

```
@@ -175,8 +175,17 @@
       "type": "string"
     },
     "src": {
-      "description": "The MF2 syntax source.",
-      "type": "string"
+      "oneOf": [
+        {
+          "description": "The MF2 syntax source.",
```

This is hard to understand? Does it mean:

```suggestion
          "description": "The source message in the MF2 format.",
```